c2pa_c_ffi/src/c_api.rs

```diff
@@ -1104,22 +1104,25 @@ pub unsafe extern "C" fn c2pa_signer_create(
     let certs = from_cstr_or_return_null!(certs);
     let tsa_url = from_cstr_option!(tsa_url);
     let context = context as *const ();
-    println!("c2pa_signer_create");
-    let c_callback = move |context: *const (), data: &[u8]| {
+
+    // Create a callback that uses the provided C callback function
+    // The callback ignores its context parameter and will use the context set on the CallbackSigner
+    let c_callback = move |callback_context: *const (), data: &[u8]| {
```

Comment:
Prevent capture of "wrong" context.

Reply:
`let context = context as *const ();` is left alone and not captured, I believe.

Reply:
Or at least it makes it clearer in my mind that there is no capture